.github/PULL_REQUEST_TEMPLATE/pull_request_template.yaml

```
@@ -0,0 +1,45 @@
+name: Pull Request
+description: Submit a Pull Request
+labels: ['change: feature request']
+assignees:
+  - '@romcal/romcal-contributors'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-should-happen
+    attributes:
+      label: What do you want to see happen?
+      value: 'I want something new!'
+    validations:
+      required: true
+  - type: dropdown
+    id: concept
+    attributes:
+      label: What concept does this feature request relate to?
+      description: What part of the project does this feature request relate to?
+      options:
+        - Infrastructure
+        - Localization
+        - Calendar Definition
+        - Discussion
+        - Documentation
+      default: 0
+    validations:
+      required: true
+  - type: textarea
+    id: code
+    attributes:
+      label: How would you like to use this feature?
+      description: Please include a code sample for your usage.
```

I don’t think this question should be in the PR template. You might want to ask how to use the new feature or something along the lines.

---

i don't see how what you're saying is different from how i'm saying it. i'll rephrase regardless.

```suggestion
      label: Demonstrate how to use the new behavior
      description: Please include a code sample for your usage.
```